evalml/automl/automl_search.py

```diff
@@ -283,7 +283,7 @@ def __init__(
                 "Time series support in evalml is still in beta, which means we are still actively building "
                 "its core features. Please be mindful of that when running search()."
             )
-
+        self._SLEEP_TIME = 0.1
```

Comment:
One of the things I found is that the sleep time increases the run time for some unit tests that run many iterations. 

This is a quick way to manipulate the amount of sleep time for some tests but it's not the only one! Let me know if you think there's a better way.

Reply:
This is the [thread](https://github.com/alteryx/evalml/pull/1975#discussion_r605132883) where we discussed the sleep in the dask pr. 

We picked 0.1 just cause it seemed like a good number. Decreasing the sleep time will not change the functionality especially since these tests use the sequential engine.